lib/src/command/uploader.dart

```diff
@@ -17,7 +17,7 @@ class UploaderCommand extends PubCommand {
   String get name => 'uploader';
   @override
   String get description =>
-      'Manage uploaders for a package on pub.dartlang.org.';
+      'Manage uploaders for a package on pub.dartlang.org. (Deprecated)';
```

Comment:
Instead of writing this in the description, I think we should make the command `hidden: true`:
https://pub.dev/documentation/args/latest/command_runner/Command/hidden.html

Then we should add it to the `CHANGELOG.md` in the SDK

Reply:
Done